scripts/exglobal_stage_ic.sh

```diff
@@ -36,16 +36,16 @@ source="${BASE_CPLIC}/${CPL_ATMIC}/${PDY}${cyc}/${CDUMP}/${CASE}/INPUT/gfs_ctrl.
 target="${COM_ATMOS_INPUT}/gfs_ctrl.nc"
 ${NCP} "${source}" "${target}"
 rc=$?
-[[ ${rc} -ne 0 ]] && error_message "${source}" "${target}" "${rc}"
-err=$((err + rc))
+(( rc != 0 )) && error_message "${source}" "${target}" "${rc}"
+(( err = err + rc ))
```

Comment:
I think @WalterKolczynski-NOAA would like us to revert this to
```suggestion
err=$((err + rc))
```